static/styles/portico/portico.css

```diff
@@ -265,7 +263,7 @@ html {
 .help .markdown {
     background-color: hsl(0, 0%, 100%);
     width: calc(100vw - 300px);
-    height: calc(100vh - 59px);
+    margin-left: 300px;
```

Comment:
The default height is removed from the .markdown as now the scroll is handled by the html, not the .markdown. This solves the issue of some wried grey patches coming.